Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.md

```diff
@@ -0,0 +1,2 @@
+##### CaseManagement-Generic
```

Comment:
Need to fix the release notes according to our template
Change the headers and the contents as follows:
```
#### Scripts
##### Script name
- Note: Moved to the *pack-name* pack.
```